cmd/admin-handlers_test.go

```diff
@@ -177,20 +177,24 @@ var (
 				"endpoint": ""
 			}
 		}
-	    },
-	    "logger": {
+	},
+	"logger": {
 		"console": {
-		    "enabled": true
+			"enabled": true
 		},
 		"http": {
-		    "1": {
-			"enabled": false,
-			"endpoint": "http://user:example@localhost:9001/api/endpoint"
-		    }
+			"target1": {
+				"enabled": false,
+				"endpoint": "https://username:password@example.com/api"
+			}
 		}
-	    }
-
-	}`)
+	},
+	"compress": {
+		"enabled": false,
+        "extensions":[".txt",".log",".csv",".json"],
```

Comment:
looks like some tabbing mistake can you indent this properly?

Reply:
Done